src/forms/Textarea.jsx

```diff
@@ -150,7 +151,6 @@ Textarea.propTypes = {
 	]),
 	labelClassName: PropTypes.string,
 	required: PropTypes.bool,
-	autoHeight: PropTypes.bool,
```

Comment:
Thought I would remove this prop because I don't see it used anywhere. Let me know if I was wrong here.